crates/forge/src/scarb.rs

```diff
@@ -165,20 +165,59 @@ pub fn dependencies_for_package(
 mod tests {
     use super::*;
     use assert_fs::fixture::{FileTouch, FileWriteStr, PathChild, PathCopy};
-    use indoc::indoc;
+    use assert_fs::TempDir;
+    use indoc::{formatdoc, indoc};
     use scarb_metadata::MetadataCommand;
     use std::process::Command;
+    use std::str::FromStr;
+
+    pub fn setup_package(package_name: &str) -> TempDir {
```

Comment:
I'm assuming this is only visible inside `tests` module anyways, but does this need to be public?